gui.js

```diff
@@ -3027,11 +3027,10 @@ IDE_Morph.prototype.importMedia = function (folderName) {
     // costumes, backgrounds or sounds
     var myself = this,
         msg = this.showMessage('Opening ' + folderName + '...');
-    this.getMediaList(
-        folderName,
-        function (items) {
-            msg.destroy();
-            myself.popupMediaImportDialog(folderName, items);
+    myself.getURL(
+        folderName + "/catalog.json",
```

Comment:
This should be:

`this.getURL(this.resourceURL(folderName, 'catalog.json'), ...`